src/batch/src/task/task_manager.rs

```diff
@@ -68,15 +72,22 @@ impl BatchManager {
                 .build()
                 .unwrap()
         };
+
+        let mem_context = Arc::new(MemoryContext::new(None, metrics.batch_total_mem.clone()));
```

Comment:
The stat in this pr only calculates the total memory instead of the memory for each batch query right?

Reply:
Currently, we only mem usage of three levels:
1. All mem used by batch tasks
2. Mem used by each batch task
3. Mem used by each executor

I don't see the necessity to add query level mem usage, but it will be quite easy.